Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2499.cs

```diff
@@ -38,6 +38,8 @@ public void Issue2499Test()
 		{
 			RunningApp.WaitForElement("picker");
 			RunningApp.Tap("picker");
+			RunningApp.WaitForElement("cat");
+
 			AppResult[] items = RunningApp.Query("cat");
```

Comment:
It's fine like this but you can also just replace the Query with WaitForElement in the future. WaitForElement also returns an array of AppResults